docs/nb/Healpix_PU.ipynb

```diff
@@ -0,0 +1,841 @@
+{
```

Comment:
<p>Why is the sum not close to 1? i.e 0.999 Is that expected?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/FRBs/astropath/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Because 0.1 of the prior is in the unseen (i.e. P_U = 0.1)</p>